site/sidebars.js

```diff
@@ -360,6 +341,20 @@ const sidebars = {
       },
       items: [...redTeamSidebar, { type: 'doc', id: 'guides/llm-redteaming' }],
     },
+    {
+      type: 'category',
+      label: 'Model scanning',
+      link: {
```

Comment:
Normally we put the index page inside the folder, would prefer that unless you feel vvvvvvvvv strongly